### ISSUE TITLE
Provider - activity events processing bug - Provider got 78 events after weekend

### ISSUE THREAD

**Issue by nieznanysprawiciel** [golemfactory/yagna#1095]
`[2021-02-28T12:29:14Z INFO  ya_provider::execution::task_runner] Collected 78 activity events. Processing...`

**Comment by tworec** [comment]
[ya-provider_r2021-02-28_00-00-02.log.zip](https://github.com/golemfactory/yagna/files/6062614/ya-provider_r2021-02-28_00-00-02.log.zip)

**Comment by tworec** [comment]
Here is `activity.db` from mbenke's computer
[mbenke-activity.db.zip](https://github.com/golemfactory/yagna/files/6069832/mbenke-activity.db.zip)

It witnesses that all 78 activity events were created within the single agreement by single requestor.
It happily coincides that this requestor owned by @mfranciszkiewicz 

```
$ echo "select distinct a.agreement_id from activity a, activity_event b where b.activity_id = a.id and b.event_date > '2021-02-28 12:23:12.453659556' and b.event_date < '2021-02-28 12:29:14';" | sqlite3 activity.db

634c5b91453cc311fce76ba7ba128fcc0b1b6eec747f5fd7fce5062b85b087e4

$ echo "select count(1) from activity a, activity_event b where b.activity_id = a.id and b.event_date > '2021-02-28 12:23:12.453659556' and b.event_date < '2021-02-28 12:29:14';" | sqlite3 activity.db

78
```

**Comment by tworec** [comment]
@mfranciszkiewicz pls paste here all logs that you have for that particular period from your requestor machine.

**Comment by tworec** [comment]
> logs that particular period from @mfranciszkiewicz requestor machine.
> [mf-requestor-blender-yapapi.zip](https://github.com/golemfactory/yagna/files/6070008/mf-requestor-blender-yapapi.zip)

In the `yapapi` debug log I can see that not only mbenkes computer was affected, but two others also (`reqc.b1` for sure).
The issue was that yapapi is using `timeout` param and activity service was expecting `pollTimeout`. It was addressed within both: #1105 and golemfactory/ya-client#102

**Comment by tworec** [comment]
closing #1109
